src/sheet_metal_conus.py

```diff
@@ -23,6 +44,12 @@ class SheetMetalConus(inkex.Effect):
     color_marker_dim = '#703cd6'    # purple
     color_marker_chords = '#9d2222' # red
     color_marker_base = '#36ba36'   # green
+    # Arrowed lines
+    dimline_style = {'stroke'        : '#000000',
+                     'stroke-width'  : '0.75px',
+                     'fill'          : 'none',
+                     'marker-start'  : 'url(#ArrowDIN-start)',
+                     'marker-end'    : 'url(#ArrowDIN-end)'    }
```

Comment:
This is how we add arrows - as a marker. Design for arrow is DIN suitable and taken from dimensions extension